README

```diff
@@ -1469,12 +1467,15 @@ CPU Core (speed up)
 
 Graphics emulation (speed up)
   VGA emulation is a demanding part of DOSBox in terms of actual CPU usage.
-  Increase the number of frames skipped (in increments of one) by pressing
-  CTRL-F8. Your CPU usage should decrease when using a fixed cycle setting,
-  and you will be able to increase cycles with CTRL-F12.
+  You can use keymapper to bind "Increase frameskip" action ("Inc Fskip"),
+  and use it to adjust skipped frames at runtime.  Your CPU usage should
+  decrease when using a fixed cycle setting, and you will be able to increase
+  cycles with CTRL-F12.
```

Comment:
On systems with limited CPU performance, consider binding a hotkey to the "Increase frameskip" action ("Inc Fskip") using the keymapper. Skipping frames generally frees up your CPU, providing faster emulation when using max cycles or affording a greater number of fixed cycles with CTRL+F12.